src/main/java/GoToJailUserCase/GoToJailInteractor.java

```diff
@@ -0,0 +1,59 @@
+package GoToJailUserCase;
+
+import MainEntities.GameBoard;
+import MainEntities.Player;
+import Tiles.GoToJailTile;
+
+public class GoToJailInteractor implements GoToJailInputBoundary{
+    final GoToJailOutputBoundary output;
+    final GameBoard board;
+    Player player;
+    GoToJailTile jailTile;
+
+
+    public GoToJailInteractor(GameBoard board, GoToJailOutputBoundary output, GoToJailTile jailTile,
+                                  Player player) {
+        this.output = output;
+        this.board = board;
+        this.jailTile = jailTile;
+        this.player = player;
+    }
+
+    /**
+     * To find the index of the jail
+     * @param board The board of the game.
+     * @param jailTile The class that contains the information that where a player will go to jail
+     */
+
+    public int jailIndex(GoToJailTile jailTile, GameBoard board){
```

Comment:
This method is unnecessary - you don't need to iterate through all the tiles. Furthermore, there is a method in GameBoard - getTileAt that can replace this.

Reply:
No, we need to know the index of the jailtile to use getTileAt, but we don't know it